packages/bpk-mixins/src/mixins/_buttons.scss

```diff
@@ -100,23 +104,31 @@
 ///   }
 
 @mixin bpk-button--secondary {
-  background-color: $bpk-button-secondary-background-color;
   background-image: $bpk-button-secondary-background-image;
-  color: $bpk-button-secondary-color;
-  box-shadow: $bpk-button-secondary-box-shadow;
+
+  @include bpk-themeable-property(color, --bpk-button-secondary-text-color, $bpk-button-secondary-color);
+  @include bpk-themeable-property(background-color, --bpk-button-secondary-background-color, $bpk-button-secondary-background-color);
+  @include bpk-border-lg($bpk-button-secondary-border-color);
+  @include bpk-border-lg(var(--bpk-button-secondary-border-color, $bpk-button-secondary-border-color));
```

Comment:
I notice it's changed from `box-shadow` to using a mixin - is this just because it should have been using the mixin in the first place?

Reply:
Yeah, this mixin still uses box-shadow under the hood!

I think the buttons pre-date the mixin.

Reply:
Thought that might be the case :)